index.html

```diff
@@ -68,6 +68,128 @@ <h2>I'am a Software Developer</h2>
         </div>
         <div class="background"></div>
       </section>
+      <section class="works">
```

Comment:
It's important that we use `display: grid` to display the cards of the projects 🤓 Please implement this change so that the project meets the requirements 😎

Reply:
I am using `display: grid` in order to display my cards. My main.css code shows that on line 309:
`.cards-wrapper {
  margin-top: 37px;
  display: grid;
  grid-template-columns: repeat(auto-fit, 360px);
  row-gap: 37px;
  column-gap: 24px;
}`